standard_schema/prerelease/HED8.3.0.mediawiki

```diff
@@ -207,33 +207,33 @@ Each term in this vocabulary has a human-readable description and may include ad
-***** Abdomen <nowiki>{hedId=HED_0000000}[The part of the body between the thorax and the pelvis.]</nowiki>
-***** Navel <nowiki>{hedId=HED_0000000}[The central mark on the abdomen created by the detachment of the umbilical cord after birth.]</nowiki>
+***** Abdomen <nowiki>[The part of the body between the thorax and the pelvis.]</nowiki>
+***** Navel <nowiki>[The central mark on the abdomen created by the detachment of the umbilical cord after birth.]</nowiki>
 ***** Waist <nowiki>{hedId=HED_0012214} [The abdominal circumference at the navel.]</nowiki>
-***** Pelvis <nowiki>{hedId=HED_0000000}[The bony structure at the base of the spine supporting the legs.]</nowiki>
-***** Pelvis-part <nowiki>{hedId=HED_0000000} [A part of the pelvis.]</nowiki>
+***** Pelvis <nowiki>[The bony structure at the base of the spine supporting the legs.]</nowiki>
+***** Pelvis-part <nowiki> [A part of the pelvis.]</nowiki>
 ****** Buttocks <nowiki>{hedId=HED_0012208} [The round fleshy parts that form the lower rear area of a human trunk.]</nowiki>
 ****** Gentalia <nowiki>{deprecatedFrom=8.1.0, hedId=HED_0012209} [The external organs of reproduction. Deprecated due to spelling error. Use Torso/Genitalia.]</nowiki>
-****** Genitalia <nowiki>{hedId=HED_0000000} [The external organs of reproduction and urination, located in the pelvic region. This includes both male and female genital structures.]</nowiki>
+****** Genitalia <nowiki> [The external organs of reproduction and urination, located in the pelvic region. This includes both male and female genital structures.]</nowiki>
 ****** Hip <nowiki>{hedId=HED_0012210} [The lateral prominence of the pelvis from the waist to the thigh.]</nowiki>
 **** Upper-extremity <nowiki>{hedId=HED_0012215} [Refers to the whole superior limb (shoulder, arm, elbow, wrist, hand).]</nowiki>
-**** Upper-extremity-part <nowiki>{hedId=HED_0000000} [A part of the upper extremity.]</nowiki>
+**** Upper-extremity-part <nowiki> [A part of the upper extremity.]</nowiki>
 ***** Upper-arm <nowiki>{hedId=HED_0012228} [Portion of arm between shoulder and elbow.]</nowiki>
-***** Upper-arm-part <nowiki>{hedId=HED_0000000} [A part of the upper arm.]</nowiki>
+***** Upper-arm-part <nowiki> [A part of the upper arm.]</nowiki>
 ****** Shoulder <nowiki>{hedId=HED_0012227} [Joint attaching upper arm to trunk.]</nowiki>
 ***** Forearm <nowiki>{hedId=HED_0012217} [Lower part of the arm between the elbow and wrist.]</nowiki>
-***** Forearm-part <nowiki>{hedId=HED_0000000} [A part of the forearm.]</nowiki>
+***** Forearm-part <nowiki> [A part of the forearm.]</nowiki>
 ****** Elbow <nowiki>{hedId=HED_0012216} [A type of hinge joint located between the forearm and upper arm.]</nowiki>
 ***** Hand <nowiki>{hedId=HED_0012218} [The distal portion of the upper extremity. It consists of the carpus, metacarpus, and digits.]</nowiki>
-***** Hand-part <nowiki>{hedId=HED_0000000} [A part of the hand.]</nowiki>
+***** Hand-part <nowiki> [A part of the hand.]</nowiki>

```

Comment:
According to Google the Hand does not include the Wrist.

Reply:
Moved one level up. Done.